lib/mysql_rewinder/cleaner.rb

```diff
@@ -31,5 +35,22 @@ def all_tables
         WHERE TABLE_SCHEMA = DATABASE()
       SQL
     end
+
+    private
+
+    def log_sql(sql)
```

Comment:
To simplify code, I think we must rename this method into `log_and_execute(sql)`, and execute the query inside this method (replace `yield` by `@client.execute(sql)`).

Reply:
Thanks. Fixed for https://github.com/DeNA/mysql_rewinder/pull/4/commits/69070ff65426744b12b767cd63df71029bed3bc5